packages/v4/patternfly-docs/content/design-guidelines/styles/icons/IconsTable.js

```diff
@@ -78,189 +64,137 @@ export class IconsTable extends React.Component {
-      : null;
-    // 2 unicodes are hard coded in icons.js, otherwise find in unicodes mapping from Core
-    const iconUnicode = Unicode || iconUnicodes.default[Name] || '';
-  
-    return {
-      removeBorder,
-      cells: [
-        {
-          title: hasIcon
-            ? (<Tooltip content="Download SVG" position={TooltipPosition.bottom}><Icon onClick={this.onDownloadSvg} color={color} /></Tooltip>)
-            : ' ',
-          props: { column: 'Icon' }
-        },
-        {
-          title: Name,
-          props: { column: 'Name' }
-        },
-        Style,
-        {
-          title: Type,
-          props: { column: 'Type' }
-        },
-        ReactName,
-        {
-          title: Label
-            ? (
-              <>
-                <div>{Contextual_usage}</div>
-                <br/>
-                <div>Tooltip: {Label}</div>
-              </>
-            )
-            : Contextual_usage,
-          props: { column: 'Usage/tooltip' }
-        },
-        {
-          title: <Tooltip content={this.state.tooltipContent} position={TooltipPosition.bottom}><span onMouseEnter={this.onHoverUnicode} onClick={this.onCopyText} color={color}>{iconUnicode}</span></Tooltip>,
-          props: { column: 'Unicode' }
-        }
-      ]
-    }
+  const clipboardCopyFunc = (event, text) => {
+    navigator.clipboard.writeText(text.toString());
+    setCopied(true);
   };
 
-  render() {
-    const { searchValue, columns, sortBy } = this.state;
-    const { direction, index } = sortBy;
-    const SearchIcon = icons.SearchIcon;
-    const searchRE = new RegExp(searchValue, 'i');
-    const iconRows = iconsData
-      .map(row => {
-        if (Array.isArray(row)) {
-          const lastIdx = row.length - 1;
-          return row.map((subRow, idx) => idx < lastIdx
-            ? this.buildRows(subRow, true)
-            : this.buildRows(subRow, false)
-          )
-        }
-        return this.buildRows(row);
-      })
-
-    let filteredRows = iconRows.filter(row => {
-      const isSearchMatch = row =>row.cells.some(cell => {
-        const searchField = typeof cell.title === 'string'
-          ? cell.title
-          : cell;
-        return searchRE.test(searchField);
-      })
-
-      return Array.isArray(row)
-        ? row.some(subRow => isSearchMatch(subRow))
-        : isSearchMatch(row)
-    })
-
-    if (direction) {
-      const sortedRows = filteredRows.sort((a, b) => {
-        a = Array.isArray(a) ? a[0] : a;
-        b = Array.isArray(b) ? b[0] : b;
-        return a.cells[index].title < b.cells[index].title
+  const filteredRows = React.useMemo(() => {
+    return iconsData.filter(icon => {
+      return icon.Name.toLowerCase().includes(searchValue.toLowerCase()) ||
+        icon.Contextual_usage?.toLowerCase().includes(searchValue.toLowerCase()) ||
+        icon.Extra_context?.toLowerCase().includes(searchValue.toLowerCase()) ||
+        icon.Label?.toLowerCase().includes(searchValue.toLowerCase()) ||
```

Comment:
Suggest including `React_name` in the search

```suggestion
        icon.Label?.toLowerCase().includes(searchValue.toLowerCase()) ||
        icon.React_name.toLowerCase().includes(searchValue.toLowerCase()) ||
```